performance-testing/performance-testing-accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/performancetesting/ingest/AccumuloElementIngestTest.java

```diff
@@ -164,10 +165,10 @@ public static void main(final String[] args) throws StoreException, IOException
         accumuloStore.initialise(testProperties.getGraphId(), schema, storeProperties);
         LOGGER.info("Initialised Accumulo store (instance name is {}, graph id is {})",
                 accumuloStore.getProperties().getInstance(),
-                accumuloStore.getProperties().getTable());
+                accumuloStore.getTableName());
         LOGGER.info("Using test properties of {}", testProperties);
         final Graph graph = new Graph.Builder()
-                .graphId(testProperties.getGraphId())
+                .config(new GraphConfig(testProperties.getGraphId()))
```

Comment:
It MAY be better to use the GraphConfig Builder as it may set some defaults

Reply:
Seems fine to me. I can't see any downside so long as those defaults don't change the overall behaviour.